frontend/packages/console-app/src/index.ts

```diff
@@ -1 +1 @@
-import '@console/internal/components/app';
+import '../../../public/components/app';
```

Comment:
This seems to circumvent a circular dependency issue.

In webpack config, Console application entry module is declared as `@console/app`

```ts
entry: ['./polyfills.js', '@console/app', 'monaco-editor-core/esm/vs/editor/editor.worker.js'],
```

so `@console/app` is the root (total 3 roots in `entry`) for module dependency graph resolution.

As @jtomasek pointed out, we should move `public/components/app.jsx` to `@console/app` package along with related code, to avoid `@console/internal` :arrow_right: `@console/app` references. We can do that as a follow-up.

@jtomasek Can you also try

```diff
-entry: ['./polyfills.js', '@console/app', 'monaco-editor-core/esm/vs/editor/editor.worker.js'],
+entry: ['./polyfills.js', './public/components/app.jsx', 'monaco-editor-core/esm/vs/editor/editor.worker.js'],
```

Reply:
Done